cookbook/security/voters.rst

```diff
@@ -190,61 +248,128 @@ and tag it with ``security.voter``:
         // app/config/services.php
         use Symfony\Component\DependencyInjection\Definition;
 
-        $definition = new Definition('AppBundle\Security\Authorization\Voter\PostVoter');
-        $definition
+        $container->register('app.post_voter', 'AppBundle\Security\Authorization\Voter\PostVoter')
```

Comment:
```diff
- $container->register('app.post_voter', 'AppBundle\Security\Authorization\Voter\PostVoter')
+ $container->register('app.post_voter', 'AppBundle\Security\PostVoter')
```

if you decide to keep it in the `AppBundle\Security` namespace. Same for Xml.
Also, `public: false` is missing from yaml.